lib/index.js

```diff
@@ -20,10 +20,28 @@ const viewer = new Viewer(renderer)
 
 let controls = new THREE.OrbitControls(viewer.camera, renderer.domElement)
 
+const fpsCheck = 1000 // ms
+let lastTime = performance.now()
+let frameCount = 0
+
+function handleUpdate () {
+  frameCount++
+  const currentTime = performance.now()
+
+  if (currentTime - lastTime > fpsCheck) {
+    const fps = Math.ceil((frameCount / ((currentTime - lastTime) / fpsCheck)))
+    lastTime = currentTime
+    frameCount = 0
+    if (fps != null) socket.emit('renderFPS', { id: socket.id, fps })
```

Comment:
Please add to doc + use it in an example to explain what it's good for

Reply:
I do not know where I would put this server-sided event (renderFPS) firing into docs as there is no mention of server-sided code in the READMEs.

I included documentation on the bot.viewer event firing, however.